frontend/README.md

```diff
@@ -49,65 +51,109 @@ WARNING: Do not store sensitive data in the environment variables.
 
 ## Deployment
 
-1.  Ensure your `awscli` is configured with the
-    [required credentials and profiles](../docs/awscli.md).
-    Set the appropriate `AWS_PROFILE`.
+1. Ensure your `awscli` is configured with the
+   [required credentials and profiles](../docs/awscli.md).
+   Set the appropriate `AWS_PROFILE`.
+
+   ```shell
+   export AWS_PROFILE=single-cell-dev
+   ```
+
+1. **Specify deployment.**
+
+   Set the `DEPLOYMENT_STAGE` environment variable to a valid deployed environment: `dev`, `staging`
+
+   ```shell
+   export DEPLOYMENT_STAGE=dev
+   ```
+
+1. **Deploy.**
+
+   Files are deployed to a publicly accessible bucket. Do not include sensitive data in the deployed files.
+
+   ```shell
+   make deploy
+   ```
+
+## Smoke Test
+
+### Update Tests
+
+#### Gatsby Dev Build
+
+1. Start FE server: `npm run develop`
+
+2. Run tests: `npm run e2e`
```

Comment:
use 1.  to auto [enumerate](http://guides.github.com/features/mastering-markdown)
```suggestion
1. Run tests: `npm run e2e`
```